### ISSUE TITLE
Inch fraction support

### ISSUE THREAD

**Issue by elquattro** [thomasa88/ParametricText#30]
Are inch fractions supported? I wanted to add both mm and inch dimensions to my model, but so far inches only show as a decimal and not a regular fraction. Fusion itself is already set to fraction mode for imperial units, but that doesn't seem to help.

**Comment by thomasa88** [comment]
Hi, there is no such formatting in ParametricText at the moment.

If you have entered an expression without any calculations, then the expression might render as inch fractions:
`{parameter.expr}`

I'm not very used to inch fractions. There might be a way to add it to the add-in. Or at least, make the add-in format numbers according to user preferences.
Can you please try this:

1. Press Ctrl+Alt+C to open the console window
2. Change the console to the "Py" option
3. Enter:
4. `adsk.core.Application.get().activeProduct.fusionUnitsManager.formatIntervalValue(2.54 * 1.5)`

Play around with the input value and see if the output is what you expect. (The internal unit is cm). Please paste the output here as well.

There is also a function to convert the value regardless of the user settings, but I have yet to find the value that should be used for fractional inch:

`adsk.core.Application.get().activeProduct.fusionUnitsManager.convert(2.54, "cm", "inch")`

**Comment by thomasa88** [comment]
Might add an extra property - maybe `.inchfrac` calculated using method described here: https://forums.autodesk.com/t5/fusion-360-api-and-scripts/convert-value-to-fractional-inch/m-p/10801689#M15083

**Comment by thomasa88** [comment]
@elquattro Please this beta release: https://github.com/thomasa88/ParametricText/releases/tag/v2.3.0-beta.1 

Use `{parameter.inchfrac}`.

Is this what you expected?

![image](https://user-images.githubusercontent.com/89997/151654705-d4434491-2f60-4bcb-9eff-11ee5c280253.png)


Some tests:
```
1.25     | 1 1/4"
2.75     | 2 3/4"
1.1      | 1 1/10"
5.0      | 5"
0.2      | 1/5"
0.0      | 0"
-1.0     | -1"
-10.3    | -10 3/10"
0.203125 | 13/64"
```

**Comment by elquattro** [comment]
Hello Thomas,
yes, this was exactly what I meant. I wasn't as successful installing the
plugin on my macbook, but i'll give it a try next week when i get back to
my desktop pc running windows.
Anyway thank you for the great work.

Best regards,
Alexander

Am Sa., 29. Jan. 2022 um 10:06 Uhr schrieb Thomas Axelsson <
***@***.***>:

**Comment by wa0uwh** [comment]
Yes, Thanks Thomasa88, I use ParametricText on ALL of my projects, Again
THANKS
Regards,
Eldon Brown

**Comment by thomasa88** [comment]
Published in Release 2.3.0. https://github.com/thomasa88/ParametricText/releases/tag/v2.3.0
Should appear in the Autodesk store in a few weeks.
